app/scripts/keyring-controller.js

```diff
@@ -73,7 +73,7 @@ module.exports = class KeyringController extends EventEmitter {
   // or accept a state-resolving promise to consume their results.
   //
   // Not all methods end with this, that might be a nice refactor.
-  fullUpdate() {
+  fullUpdate () {
```

Comment:
If you want to make this a rule, you should just make it a hard fail in the `.eslintrc` file, I feel bad with you always patching up the warnings alone.